src/index.html

```diff
@@ -1,12 +1,426 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="/">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             class="nav__item-link is-active"
+             href="/apple"
+            >
+             Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/samsung">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/smartphones">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__item-link"
+              href="/laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/gadgets">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/tablets">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/photo">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item-link" href="/video">
+              Video
+            </a>
+          </li>
+        </ul>
+
+      </nav>
+    </header>
+    <main>
+      <section class="catalog">
+        <div class="catalog__container">
+          <article class="card catalog__card" data-qa="card">
+            <a class="card__link">
+              <img
+                class="card__link-img"
+                src="images/imac.jpeg"
+                alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+              >
+            </a>
+            <h3 href="/" class="card__name">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
Check indents for the content

Reply:
Indents for the content: it should contain 2 spaces, you have more here
```suggestion
              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```